### ISSUE TITLE
Seperate PF and PM FMGS Screens

### ISSUE THREAD

**Issue by ecortinas** [flybywiresim/a32nx#7433]
### Aircraft Version

Development

### Description

Any change that the MCDU screen will eventually be independent so I can have on screen on pilot side and a different screen on co-pilot side?

**Comment by Benjozork** [comment]
This will be done because it exists in the real plane (therefore it isn't necessary to request it). However as every feature it that isn't yet available, other things need to happen beforehand and a lot of work is involved.

**Comment by 2hwk** [comment]
Duplicate of https://github.com/flybywiresim/a32nx/issues/1058 closing.
